model/registry/cloudfoundry.yaml

```diff
@@ -0,0 +1,158 @@
+groups:
+  - id: registry.cloudfoundry
+    prefix: cloudfoundry
+    type: attribute_group
+    brief: >
+      CloudFoundry resource attributes.
+    attributes:
+      - id: system.id
+        type: string
+        stability: experimental
+        brief: >
+          A guid or another name describing the event source.
+        note: |
+          CloudFoundry defines the `source_id` in the [Loggegator v2 envelope](https://github.com/cloudfoundry/loggregator-api#v2-envelope).
+          It is used for logs and metrics emitted by CloudFoundry. It is
+          supposed to contain the component name, e.g. "gorouter", for
+          CloudFoundry components.
+
+          When system components are instrumented, values from the
+          [Bosh spec](https://bosh.io/docs/jobs/#properties-spec)
+          should be used. The `system.id` should be set to
+          `spec.deployment/spec.name`.
+        examples:
+          - 'cf/gorouter'
+      - id: system.instance.id
```

Comment:
any reason  not to reuse existing attribute `service.instance.id` ? see https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/service.md

Reply:
One more thing: Eventually, you would want to have an attribute to identify the CloudFoundry deployment as a whole, to distinguish between different installations. Maybe that would be a proper use for `service.namespace`,